src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -43,7 +43,7 @@
     <Delete Files="%(_ReferenceSerializerIntermediateFolder.Identity).cs" ContinueOnError="true"/>
     <Delete Files="%(_ReferenceSerializerIntermediateFolder.Identity).pdb" ContinueOnError="true"/>
     <Message Text="Running Serialization Tool for Reference Assembly" Importance="normal" />
-    <Exec Command="dotnet Microsoft.XmlSerializer.Generator --force --quiet --reference @(Reference) --assembly %(_TargetSerializationAssembly.Identity) --type %(_TargetSerializationAssembly.SerializationTypes) --out $(IntermediateOutputPath)" ContinueOnError="true" />
+    <Exec Command="dotnet Microsoft.XmlSerializer.Generator --force --quiet --reference &quot;@(Reference)&quot; --assembly &quot;%(_TargetSerializationAssembly.Identity)&quot; --type %(_TargetSerializationAssembly.SerializationTypes) --out $(IntermediateOutputPath)" ContinueOnError="true" />
```

Comment:
You also need quotes around the IntermediateOutputPath as that directory might contain spaces.

Reply:
When quote the immediateoutputpath, it will add a quote in the end of the path e.g. "obj\Debug\netcoreapp2.0\", the path parsed in will be obj\Debug\netcoreapp2.0\" So it will return error directory not exist. I will still add quote for it but need update our code a little to handle this case.